### PR TITLE
JavaScript Engineer - Answers to the Questions

### PR DESCRIPTION
**Pull request by bearnithi**
Hi, This is Nithya Rajan. Applying for JavaScript Engineer.

### CODE REVIEW

File: answers.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+## The real stuff.
+
+1. Which all programming languages are installed on your system.
+   Node JS, Angular, TypeScript
+
+2. Write a function that takes a number and returns a list of its digits in an array.
+
+   ```javascript
+   function printNum(number) {
+     for (let i = 0; i <= number; i++) {
```
Comment (niksmac) [inline answers.md]: `takes a number and returns a list of its digits in an array` Please double check.

**Review by niksmac — CHANGES_REQUESTED** [review]
Please check question 1 once again.

File: answers.md (hunk 1 above)
Comment (bearnithi) [reply answers.md]: so you mean, it should return an array which contains the list of numbers from 0 to the given parameter.

Comment (bearnithi) [reply answers.md]: updated the function. please check it and let me know if I have to change anything.

Comment (niksmac) [reply answers.md]: I mean if I call `theFn(123)` i should get `[1,2,3]` as output.

**Pull request closed without merging**